dali/driver/uart.py

```diff
@@ -0,0 +1,109 @@
+
+"""
+UART communication driver for DALI
+"""
+
+import logging
+import serial
+import struct
+from dali.command import Command
+from dali.interface import DriverInterface, ParameterError
+
+
+class DaliUART(DriverInterface):
```

Comment:
It has no specific name, its a raw 8-bit protocoll. The message heads with a length, and trailing with the crc of the message. Thats it.